test/test.js

```diff
@@ -323,6 +323,63 @@ describe('csurf', function () {
     })
   })
 
+  describe.only('req.verifyToken()', function () {
+    var app, token, cookie;
+    beforeEach(function(done) {
+      app = connect()
+      app.use(session({ keys: ['a', 'b'] }))
+      app.use(bodyParser.urlencoded({ extended: false }))
```

Comment:
Does this test app need a body parser? I don't think I see any requests with a token in the body.